<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>16</y>
      <width>795</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actionColour"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionItalic"/>
    <addaction name="actionBold"/>
    <addaction name="actionWordtype"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>对齐</string>
    </property>
    <addaction name="actionLift"/>
    <addaction name="actionRight"/>
    <addaction name="actionMiddle"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>背景</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionSearch"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew_N">
   <property name="text">
    <string>new(&amp;N)</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>saveAs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>cut</string>
   </property>
  </action>
  <action name="actionColour">
   <property name="text">
    <string>colour</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="text">
    <string>underline</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="text">
    <string>italic</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="text">
    <string>bold</string>
   </property>
  </action>
  <action name="actionWordtype">
   <property name="text">
    <string>wordtype</string>
   </property>
  </action>
  <action name="actionLift">
   <property name="text">
    <string>lift</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>right</string>
   </property>
  </action>
  <action name="actionMiddle">
   <property name="text">
    <string>middle</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
